test/libsolidity/syntaxTests/types/mapping/mapping_dynamic_key.sol

```diff
@@ -0,0 +1,5 @@
+contract c {
+	mapping(string => uint) data;
+}
+// ----
+// TypeError: (17-45): Dynamically-sized mapping keys not yet implemented.
```

Comment:
Wait a second, aren't strings working in mapping? When is that assertion hit currently?

Reply:
In `ExpressionCompiler::appendStateVariableAccessor(VariableDeclaration const& _varDecl)`

Reply:
Since the assertion fails in `appendStateVariableAccessor`, it looks like the accessor is what isn't implemented for that case.